tests/Collection/CollectionFunctionalTest.php

```diff
@@ -132,11 +132,15 @@ public function testAggregateWithinTransaction(): void
 
     public function testCreateIndexSplitsCommandOptions(): void
     {
+        $this->skipIfServerVersion('<', '4.4', 'commitQuorum and comment options are not supported');
```

Comment:
I'll note that this removes test coverage for older versions for splitting, but I don't think it makes any difference. Fine to leave as is.